sdk/compute/arm-compute/test/compute_examples.ts

```diff
@@ -0,0 +1,275 @@
+              network_name +
+              "/subnets/" +
+              subnet_name,
+          },
+        },
+      ],
+    };
+    const nic_info = await network_client.networkInterfaces.beginCreateOrUpdateAndWait(
+      group_name,
+      nic_name,
+      parameter
+    );
+  }
+
+  it("availabilitySets create test", async function() {
+    const res = await client.availabilitySets.createOrUpdate(resourceGroupName, availabilitySetName, {
+      platformFaultDomainCount: 2,
+      platformUpdateDomainCount: 20,
+      location: location,
+    })
+    assert.equal(res.name,availabilitySetName);
+  });
+
+  it("availabilitySets update test", async function() {
+    const res = await client.availabilitySets.update(resourceGroupName, availabilitySetName, {
+      platformFaultDomainCount: 2,
+      platformUpdateDomainCount: 20
+    })
+    assert.equal(res.type,"Microsoft.Compute/availabilitySets");
+  });
+
+  it("availabilitySets get test", async function() {
+    const res = await client.availabilitySets.get(resourceGroupName, availabilitySetName);
+    assert.equal(res.name,availabilitySetName);
+  });
+
+  it("availabilitySets list test", async function() {
+    const resArray = new Array();
+    for await (const item of client.availabilitySets.list(resourceGroupName)) {
+      resArray.push(item);
+    }
+    assert.equal(resArray.length,1);
+  });
+
+  it("availabilitySets delete test", async function() {
+    const res = await client.availabilitySets.delete(resourceGroupName,availabilitySetName);
+    const resArray = new Array();
+    for await (const item of client.availabilitySets.list(resourceGroupName)) {
+      resArray.push(item);
+    }
+    assert.equal(resArray.length,0);
+  });
+
+  it("virtualMachines create test", async function() {
+    await createVirtualNetwork();
+    await createNetworkInterface(resourceGroupName, location, interface_name);
+    const res = await client.virtualMachines.beginCreateOrUpdateAndWait(resourceGroupName,virtual_machine_name,{
+      location: location,
+      hardwareProfile: {
+        vmSize: "Standard_D2_v2",
+      },
+      storageProfile: {
+        imageReference: {
+          sku: "2016-Datacenter",
+          publisher: "MicrosoftWindowsServer",
+          version: "latest",
+          offer: "WindowsServer",
+        },
+        osDisk: {
+          caching: "ReadWrite",
+          managedDisk: {
+            storageAccountType: "Standard_LRS",
+          },
+          name: "myVMosdisk",
+          createOption: "FromImage",
+        },
+        dataDisks: [
+          {
+            diskSizeGB: 1023,
+            createOption: "Empty",
+            lun: 0,
+          },
+          {
+            diskSizeGB: 1023,
+            createOption: "Empty",
+            lun: 1,
+          },
+        ],
+      },
+      osProfile: {
+        adminUsername: "testuser",
+        computerName: "myVM",
+        adminPassword: "Aa!1()-xyz",
```

Comment:
@colawwj @qiaozha is this password real and needed for running the test against the service, or just some random fake value?

Reply:
some random fake value. Has we enabled credential scan in this repo ?

Reply:
@qiaozha yes, but currently it doesn't block PRs.  I will open a PR to replace this with a placeholder.  Thanks for the confirmation!